src/cityreader/cityreader.py

```diff
@@ -6,6 +6,8 @@ def __init__(self, name, lat, lon):
     self.name = name
     self.lat = lat
```

Comment:
It may be a good idea to define lat and lon as floats within the constructor rather than having to do it anytime you want to pass data into the constructor.